frame/nfts/src/lib.rs

```diff
@@ -1262,20 +1272,35 @@ pub mod pallet {
 			key: BoundedVec<u8, T::KeyLimit>,
 			value: BoundedVec<u8, T::ValueLimit>,
 		) -> DispatchResult {
+			ensure!(
+				Self::is_pallet_feature_disabled(PalletFeature::NoAttributes),
```

Comment:
Shouldn't features be always positive like rust features? I.e. the feature should be Attributes - it could be a default feature.

Reply:
This is the same thing as Oliver's point above.